### PR TITLE
Fixing tls certificates generation script and local docker multiarch

### PR DESCRIPTION
**Pull request by tock-ibm**
Signed-off-by: Gennady Laventman <gennady@il.ibm.com>
Signed-off-by: Yoav Tock <tock@il.ibm.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -99,7 +99,8 @@ test-coverage: test-coverage-tools
...
-.PHONY: docker-arm
-docker-arm:
-	$(DOCKER) buildx build --platform linux/amd64 -f $(DOCKERFILE) -t $(DOCKER_IMAGE):linux-arm64 --load .
-	$(DOCKER) buildx build --platform linux/arm/v7 -f $(DOCKERFILE) -t $(DOCKER_IMAGE):linux-arm-v7 --load .
\ No newline at end of file
+.PHONY: docker-multiarch
+docker-multiarch:
+	$(DOCKER) buildx build --platform linux/arm64 -f $(DOCKERFILE) -t $(DOCKER_IMAGE):linux-arm64 --load .
+	$(DOCKER) buildx build --platform linux/arm/v7 -f $(DOCKERFILE) -t $(DOCKER_IMAGE):linux-arm-v7 --load .
+	$(DOCKER) buildx build --platform linux/s390x -f $(DOCKERFILE) -t $(DOCKER_IMAGE):linux-s390x --load .
```
Comment (cendhu) [inline Makefile:106]: Maybe we can have `docker-x86`, `docker-arm64`, `docker-armv7`, `docker-x390x` along with `docker-multiarch`.

File: scripts/tlsGen.sh [hunk 2]
```diff
@@ -3,13 +3,14 @@ create_pki() {
         mkdir -p "$BASE_DIR/tls/$1"
 
         echo "Generating private key for $1"
-        docker run -it --rm -v $BASE_DIR/tls:/export frapsoft/openssl ecparam -name prime256v1 -genkey -noout -out "/export/$1/$1.key"
+        docker run -it --rm -v $BASE_DIR/tls:/export nginx openssl ecparam -name prime256v1 -genkey -noout -out "/export/$1/$1.key"
```
Comment (cendhu) [inline scripts/tlsGen.sh:6]: why are we moving to nginx docker image from https://hub.docker.com/r/frapsoft/openssl?

Comment (tock-ibm) [reply scripts/tlsGen.sh:6]: Gennady discovered that `frapsoft/openssl` does not support all the architectures we need. `nginx` does...

File: Makefile (hunk 1 above)
Comment (tock-ibm) [reply Makefile]: done

**Review by cendhu — APPROVED** [review]

**Pull request merged**